### PR TITLE
ManipulationHelpers extension, first commit

### PR DESCRIPTION
**Pull request by nockawa**

### CODE REVIEW

File: ManipulationHelpers/ManipulationHelpers.d.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+        private _yArrowColor;
+        private _zArrowColor;
+        private _xyPlaneSelectionColor;
+        private _xzPlaneSelectionColor;
+        private _yzPlaneSelectionColor;
+        private _highlighted;
+    }
+}
+import Vector4 = BABYLON.Vector2;
+declare module ManipulationHelpers {
```
Comment (RaananW) [inline ManipulationHelpers/ManipulationHelpers.d.ts]: just a note - no need to declare the module so many times

File: ManipulationHelpers/ManipulationHelpers.d.ts [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+        /**
+         * Display all rotation cylinders
+         */
+        Rotations = 1792,
+    }
+    /**
+     * This class create the visual geometry to display a manipulation radix in a viewport.
+     * It also implements the logic to handler intersection, hover on feature.
+     */
+    class Radix {
```
Comment (RaananW) [inline ManipulationHelpers/ManipulationHelpers.d.ts]: isn't this ( and the rest of the classes in this file) actually an interface?

Comment (nockawa) [inline ManipulationHelpers/ManipulationHelpers.d.ts]: @RaananW this is the generated definition file you're looking at right?
that's why there're multiple module declarations and no implem, correct?

Comment (RaananW) [inline ManipulationHelpers/ManipulationHelpers.d.ts]: That was a silly comment. Ignore that. I don't notice it was the .d file, and was wondering how it compiles...

**Pull request merged**